adapters/platform_mbed.cpp

```diff
@@ -64,8 +64,10 @@ const IO_INTERFACE_DESCRIPTION* platform_get_default_tlsio(void)
     return tlsio_wolfssl_get_interface_description();
 }
 
-STRING_HANDLE platform_get_platform_info(void)
+STRING_HANDLE platform_get_platform_info(PLATFORM_INFO_OPTION options)
 {
+    (void)options;
```

Comment:
same thing for each platform file where we are now passing in options but doing nothing with it.